Tests/PubNubTests/Networking/Routers/ChannelGroupEndpointTests.swift

```diff
@@ -202,29 +200,29 @@ final class ChannelGroupsEndpointTests: XCTestCase {
 
     wait(for: [expectation], timeout: 1.0)
   }
+}
 
-  // MARK: - Add Group Channel
+// MARK: - Add Channel to Group
 
-  func testGroupChannelsAdd_Endpoint() {
-    let endpoint = Endpoint.addChannelsForGroup(group: testGroupName, channels: testChannels)
+extension ChannelGroupsRouterTests {
+  func test_AddChannelsForGroup_Router() {
+    let router = ChannelGroupsRouter(.addChannelsToGroup(group: testGroupName, channels: testChannels), configuration: config)
 
-    XCTAssertEqual(endpoint.description, "Group Channels Add")
-    XCTAssertEqual(endpoint.category, .addChannelsForGroup)
-    XCTAssertEqual(endpoint.operationCategory, .channelGroup)
-    XCTAssertNil(endpoint.validationError)
+    XCTAssertEqual(router.endpoint.description, "Group Channels Add")
+    XCTAssertEqual(router.category, "Group Channels Add")
+    XCTAssertEqual(router.service, .channelGroup)
   }
 
-  func testGroupChannelAdd_Endpoint_ValidationError() {
-    let endpoint = Endpoint.addChannelsForGroup(group: "", channels: [])
+  func test_AddChannelsForGroup_Router_ValidationError() {
+    let router = ChannelGroupsRouter(.addChannelsToGroup(group: "", channels: testChannels), configuration: config)
 
-    XCTAssertNotEqual(endpoint.validationError?.pubNubError, PubNubError(.invalidEndpointType, endpoint: endpoint))
-  }
+    XCTAssertEqual(router.validationError?.pubNubError?.details.first,
+                   ErrorDescription.emptyGroupString)
 
-  func testAddChannelsForGroup_Endpoint_AssociatedValues() {
-    let endpoint = Endpoint.addChannelsForGroup(group: "GroupName", channels: ["Channels"])
+    let emptyChannel = ChannelGroupsRouter(.addChannelsToGroup(group: testGroupName, channels: []), configuration: config)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line should be 120 characters or less: currently 122 characters](https://app.codacy.com/app/PubNub/swift/pullRequest?prid=4513555)